data/campaigns/Heir_To_The_Throne/utils/abilities.cfg

```diff
@@ -104,6 +104,12 @@ _"This unit's grasp of melee tactics allows adjacent allies to strike the first
         {FOREACH units_lacking_initiative i}
             {INITIATIVE_OBJECT x,y=$units_lacking_initiative[$i].x,$units_lacking_initiative[$i].y}
         {NEXT i}
+        [lua] # Can't see why {INITIATIVE_OBJECT} leaves the last unit selected.
+            code=<<
+            wesnoth.deselect_hex()
```

Comment:
Well, it's 100% reproducible in any of those three scenario (if someone who knows the bowels of the engine wants to got hunting). And it's definitely in the INITIATIVE_OBJECT macro. To reproduce it, you need simply begin the scenario. It only occurs prior to the first turn and effects Li'sar, and troops on her size and, when she and Konrad join forces in S16, Konrad's side, as well.